pom.xml

```diff
@@ -73,6 +65,11 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>r05</version>
+        </dependency>
```

Comment:
this package is not used anywhere ??

Reply:
This thing used in WFLogger class . I removed the Cassandra dependencies that's why  "com.google.common.base.Throwables;"  this package giving error.